### PR TITLE
Toggle icons between eye and eye-slash in password show option.

### LINKED ISSUE #1822: Toggle icon between "eye" and "slash-eye" in setting password form.
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

**Current behaviour:**

Head to sign up page, and enter password, and click on eye.
**Expected behaviour:**

It should be set by default to closed, then clicking on it should make it open.

**Steps to reproduce:**
Head to sign up page, and enter password, and click on eye.

_I am working on it_

### PR DESCRIPTION
**Pull request by rupav**
Fixes #1822 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

screenshot from 2018-10-18 18-46-03
screenshot from 2018-10-18 18-46-26
screenshot from 2018-10-18 18-46-09

### CODE REVIEW

File: api/config/config.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 DEBUG = True
 
 # Environment for working, can be LOCAL/PROD
-ENV = 'PROD'
+ENV = 'LOCAL'
```
Comment (rajvaibhavdubey) [inline api/config/config.py]: Please don't make changes here.

Comment (rajvaibhavdubey) [inline api/config/config.py]: Please don't make changes here

Comment (rupav) [reply api/config/config.py]: just reverting it back 😅👍

**Comment by rupav** [comment]
@yashLadha @gabru-md @ParthS007 please review.

**Pull request merged**